concurrent_test.go

```diff
@@ -106,34 +116,56 @@ func concurrentReadAndWrite(t *testing.T,
 	minWriteBytes, maxWriteBytes int,
 	testDuration time.Duration) {
 
-	// prepare the db
+	t.Log("Preparing db.")
 	db := btesting.MustCreateDB(t)
 	err := db.Update(func(tx *bolt.Tx) error {
 		_, err := tx.CreateBucket(bucket)
 		return err
 	})
 	require.NoError(t, err)
 
+	t.Log("Starting workers.")
+	records := runWorkers(t,
+		db, bucket, keys,
+		readerCount, minReadInterval, maxReadInterval,
+		minWriteInterval, maxWriteInterval, minWriteBytes, maxWriteBytes,
+		testDuration)
```

Comment:
Can you create a configuration struct that will collect all the variables to ensure they are passed by name not by the order?

Reply:
agreed. Let me enhance this separately. thx

Reply:
resolved in https://github.com/etcd-io/bbolt/pull/463